app/Http/Controllers/AssignUserController.php

```diff
@@ -47,4 +61,41 @@ public function assign(Request $request, $todoId)
 
         // return response()->json($response);
     }
+    public function remove(Request $request, $todoId)
```

Comment:
@vrrayz Its like you are not aware  of  the current structure of the Todo, Please contact @Ericogia  to give you the updated structure, we only have a collaborators array in which we add users

Reply:
> I just checked @chixmxkx already worked and submitted this and her sumission has been approved. Sorry man

She worked on removing users from a Todo. I added the option of removing teams from the Todo. Check it well